src/style.css

```
@@ -0,0 +1,109 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 400;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.form {
+  width: 100%;
+  position: relative;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-color: #fff;
+  margin-top: 20px;
+}
+
+.form:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.label-field {
+  display: block;
+  width: 100%;
+}
+
+.label-field-big-size {
+  height: 68px;
+}
+
+.label-field-small-size {
```

```suggestion
.label-field-small {
```
думаю и так понятно, что речь идет о размере